source/Tests/Playnite.Tests/SearchViewModelTests.cs

```diff
@@ -28,6 +28,16 @@ public void MatchTextFilterTest()
             Assert.IsTrue(SearchViewModel.MatchTextFilter(null, null));
             Assert.IsTrue(SearchViewModel.MatchTextFilter("", null));
             Assert.IsTrue(SearchViewModel.MatchTextFilter(null, ""));
+
+            // Acronym tests
+            Assert.IsTrue(SearchViewModel.MatchTextFilter("gow", "Gears of War", true));

```

Comment:
These tests should include strings with random additional spaces in both arguments (`g OW` and ` god   of war 3 ` for example) and also tests with acronym search disabled.